Gemfile

```diff
@@ -45,6 +45,8 @@ group :test do
   gem 'webmock', require: false
   gem 'vcr'
   gem 'puffing-billy'
+  gem 'simplecov', require: false
+  gem 'coveralls', require: false
```

Comment:
And by run, I should say that they're starting to track files and evaluate their coverage.